packages/dds/tree/src/feature-libraries/sequence-field/rebase.ts

```diff
@@ -410,6 +411,11 @@ function rebaseMark<TNodeChange>(
 	moveEffects: MoveEffectTable<TNodeChange>,
 ): CellSpanningMark<TNodeChange> {
 	if (isSkipMark(baseMark) || isSkipLikeReattach(baseMark) || isSkipLikeDetach(baseMark)) {
+		// TODO: Rebase currMark's changes over baseMark's changes.
+		if (isObjMark(currMark) && currMark.type === "Modify") {
+			const childChange = rebaseChild(currMark.changes, undefined);
```

Comment:
Something like that, right?
```suggestion
			const childChange = rebaseChild(currMark.changes, isObjMark(baseMark) && baseMark.type !== "ReturnTo" ? baseMark.changes : undefined);
```

Reply:
Although seeing how many similar cases need to be handled, it's probably best to make a function that takes any curr and base marks and calls rebaseChild with the appropriate child changes.

Reply:
Agreed